datumaro/datumaro/util/__init__.py

```diff
@@ -17,4 +18,28 @@ def dir_items(path, ext, truncate_ext=False):
             if truncate_ext:
                 f = f[:ext_pos]
             items.append(f)
-    return items
\ No newline at end of file
+    return items
+
+def split_path(path):
```

Comment:
I prefer something like code below:

```
def split_path(p):
    head, tail = os.path.split(p)
    return split_path(head) + [tail] if tail else [head]
```

Reply:
Nice variant, but it should probably include `normpath` call too, and have some handling for 1-element paths like ".." as it gives ['', '..'].